packages/viewport/src/index.ts

```diff
@@ -65,9 +66,7 @@ function addListenerFunctions(
 	};
 }
 
-function createMediaQueryList(
-	args: undefined | { min?: number; max?: number }
-): false | MinimalMediaQueryList | MediaQueryList {
+function createMediaQueryList( args: undefined | { min?: number; max?: number } ): QueryItem {
```

Comment:
I think that `undefined` in an argument is equivalent to having an optional argument

```ts
function createMediaQueryList( args?: Partial<{min: number, max: number}> ): QueryItem {
	…
}
```

there might be an impact at the call-site if you try to pass _the value `undefined`_ vs. by _passing nothing_ but if it's happy then the optional argument would probably be more straightforward